data/human/free worlds 0 prologue.txt

```diff
@@ -1288,18 +1288,25 @@ mission "FW Katya 1"
 			
 			label karate
 			`	You dodge sideways and spin quickly. As you do so you hear a loud click, but the gun does not fire. With one hand you knock aside the gun. With the other you land a karate chop which, if you had studied karate, would've shattered her collarbone and left her reeling in pain. But since you have not studied karate, instead your blow bounces harmlessly off her surprisingly muscular shoulder.`
-			`	Your assailant turns out to be wearing a Free Worlds uniform and a disappointed expression. "Well aren't you a dumb reckless bastard!" she says, "I'd hoped for something better from you. First of all, if this gun were loaded, your brains would be splattered all over the sidewalk. And second, you fight like a girl. I have no use for hotheads like you." She stalks off, leaving you very confused. It would appear that you've just failed some sort of test.`
+			`	Your assailant turns out to be wearing a Free Worlds uniform and a very disappointed expression. "Well, aren't you a dumb, reckless bastard." she says, "I'd hoped for something better from you. If this gun was loaded, your brains would already be splattered all over the sidewalk. And you hit like a paper tiger in that's been soaking in the rain for the past few days. I have no use for hotheads like you." She makes to stalk off, leaving you very confused. It would appear that you've just failed some sort of test.`
```

Comment:
paper tiger that's been soaking

The in is unnecessary

Also the were to was change is unnecessary, both forms are valid

Reply:
If we wanted to use were, it should be like "Were this gun loaded" instead of "If this gun were loaded"

Reply:
No, here's a grammar explanation for you: https://www.grammarly.com/blog/conditional-sentences-was-instead-of-were/

To me, the sentence is an unreal conditional (the gun is NOT loaded in reality, she is telling you what would happen if it were)

Reply:
"If .. were .." Sounds bad, so I'll just use "Were .." instead, then.